libs/SmartSync/src/com/salesforce/androidsdk/smartsync/model/SalesforceObject.java

```diff
@@ -36,9 +36,7 @@
  * This class represents a typical Salesforce object.
  *
  * @author bhariharan
- * @deprecated Will be removed in Mobile SDK 7.0.
```

Comment:
I had accidentally deprecated this class. This is meant to stick around. Only `SalesforceObjectType` goes away.